languages/javascript/exercises/concept/booleans/.docs/instructions.md

```diff
@@ -0,0 +1,56 @@
+- Fast attack: you can try a fast attack on the group if the knight is sleeping because it takes time for him to get his armor on, so he will be vulnerable.
+- Spy: you can spy on the group if at least one of them is awake. Otherwise, spying is a waste of time.
+- Signal Prisoner: you can try to communicate with the prisoner by making bird sounds only if the archer is sleeping and the prisoner is awake. The reason is that a lot of archers are trained in bird signling so he could intercept your message.
+- Release Prisoner: You can sneak in and release the prisoner if he is awake and the other two characters are sleeping. Another way for achieving this would be if the archer is sleeping and Annalyn has her pet dog with him. In this case the release of the prisoner would look like this: The knight will be scared by the dog, so he will withdraw, the archer would not equip himself fast enough in order to attempt a ranged attack, so he will also have to withdraw and then the main character would be able to release the prisoner.
+
+## Tasks
+
+### 1. Check if the 'Fast Attack' action is possible
+
+Implement a function named `canExecuteFastAttack` that takes a boolean value which indicates if the knight is awake. This function returns `true` if the 'Fast Attack' action is available based on the state of the character. Otherwise, returns `false`:
+
+```javascript
+const knightIsAwake = true;
+canExecuteFastAttack(knightIsAwake)
+// => false
+```
+
+### 2. Check if the 'Spy' action is possible
+
+Implement a function named `canSpy` that takes three boolean values, indicationg if the knight, archer and the prisoner, respectively, are awake. The function returns `true` if the 'Spy' action is available based on the state of the characters. Otherwise, returns `false`:
+
+```javascript
+const knightIsAwake = false;
+const archerIsAwake = true;
+const prisonerIsAwake = false;
+canSpy(knightIsAwake, archerIsAwake, prisonerIsAwake)
+// => true
+```
+
+### 3. Check if the 'Signal Prisoner' action is possible
+
+Implement a function named `canSignalPrisoner` that takes two boolean values, indicationg if the archer and the prisoner, respectively, are awake. The function that returns `true` if the 'Signal Prisoner' action is available based on the state of the characters. Otherwise, returns `false`:
+
+```javascript
+const archerIsAwake = false;
+const prisonerIsAwake = true;
+canSignalPrisoner(archerIsAwake, prisonerIsAwake)
+// => true
+```
+
+### 4. Check if the 'Release Prisoner' action is possible
+
+Implement a function named `canReleasePrisoner` that takes four boolean values. The first three parameters indicate if the knight, archer and the prisoner, respectively, are awake. The last parameters indicates if Annalyn's pet dog is present. The function returns `true` if the 'Release Prisoner' action is available based on the state of the characters and also based on Annalyn's pet dog presence. Otherwise, returns `false`:
```

Comment:
```suggestion
Implement a function named `canReleasePrisoner` that takes four boolean values. The first three parameters indicate if the knight, archer and the prisoner, respectively, are awake. The last parameter indicates if Annalyn's pet dog is present. The function returns `true` if the 'Release Prisoner' action is available based on the state of the characters and Annalyn's pet dog presence. Otherwise, it returns `false`:
```